api/sql/schema/750$transfer.push.confirmLedger.sql

```diff
@@ -14,3 +17,16 @@ WHERE
     ledgerTxState = 1
 
 IF @@ROWCOUNT <> 1 RAISERROR('transfer.confirmLedger', 16, 1);
+
```

Comment:
raise should be last statement, as we do not want to lose the event
see pattern in other similar procedures